articles/embedded.md

```diff
@@ -74,6 +76,8 @@ To answer these questions, the OFERA EU project as well as several others have a
 ### ROS2-based approaches
 *   [**EU project OFERA**](http://ofera.eu/): The EU project OFERA (Open Framework for Embedded Robot Applications) aims at a ROS 2-compatible stack for MCUs in the range of STM32F4 or STM32L1, i.e. with possibly less than 100kB RAM. The project partners currently investigate of using the ROS 2 rmw, rcl and rclcpp layers as-is on the Micro XRCE-DDS implementation of the upcoming XRCE-DDS standard. In parallel, a more modular approach in the style of rosserial is investigated. In the project's use-cases, NuttX is considered as primary choice for the RTOS. Beyond the project page, additional details of the project results can be found at [microros.github.io/micro-ROS/](https://microros.github.io/micro-ROS/).
 
+* [Hardware Robot Operating System (H-ROS)](https://acutronicrobotics.com/modularity/H-ROS/) is an a standardized software and hardware infrastructure to create modular robot hardware. H-ROS is actively being used within the OFERA EU project to benchmark and prototype the capabilities of the ROS 2 stack against the ROS 2 embedded stack. In addition, H-ROS implements selected components of the ROS 2.0 stack for microcontrollers.
```

Comment:
> to benchmark and prototype the capabilities of the ROS 2 stack against the ROS 2 embedded stack

I am not sure about this sentence - how we can actually do it. The use-case compares H-ROS against micro-ROS, which is a valuable thing (but something else than implied by this sentence, I think).

Reply:
Hello @adamdbrw, beyond the use case, we're introducing pieces of the embedded ROS 2 stack we're building into [our SoM](https://acutronicrobotics.com/modularity/som/). This is indeed different from the use case itself but the sentence, IMHO is valid based on the capabilities embedded into the H-ROS SoM.

Let me know if this is valid for you or you need further clarifications. Happy to state this in the text if you consider it necessary. I was just trying to be brief.

Reply:
It's ok for me, thank you!